### ISSUE TITLE
Prepare for setup assigned OpenStack Clusters for use by the team

### ISSUE THREAD

**Issue by mohammedzee1000** [openshift/odo#1799]
[kind/Enhancement]

## Which functionality do you think we should update/improve?
We have recently been assigned a set of OpenStack resources internally
cluster information will be present in https://docs.google.com/document/d/1CryQefJbS317xAwZcF48jsSNpQuSMYd_E4p0Y9Rdm-8/edit?usp=sharing

### Acceptance Criteria

- [ ] Getting a service account (Optional for CI, not needed for manual testing as login is also tied to team kerberos)
- [x] planning the next step
- [x] research

## Why is this needed? 
Having static clusters should enable us to test more quickly. Also, these resources should provide us with windows machines for testing

**Comment by mohammedzee1000** [comment]
Gathering information for setting things up (dynamically updated, private information, if any will be in the doc linked)

## Openshift Clusters on Openstack

### Ansible route (<=3.11)

#### Preparing nodes:
https://github.com/openshift/openshift-ansible-contrib/tree/master/playbooks/provisioning/openstack

#### Cluster setup

Use well known openshift-ansible

### Installer route (>=4.0)

#### Node setup

https://github.com/openshift/installer/blob/master/docs/user/openstack/README.md

**Comment by kadel** [comment]
Does this mean that we will have two static clusters (one 3.11, and one 4.x)? That would be great!

**Comment by mohammedzee1000** [comment]
Yes, that is correct. With current limits, we can allocate 40 individual machines, dividing the rest of the resources between them, as needed.

**Comment by mohammedzee1000** [comment]
Semi blocker: https://jira.coreos.com/projects/DPTP/issues/DPTP-406?filter=allopenissues
Required to implement as part of CI, but will not block preparation and research work or even manual usage.

**Comment by mohammedzee1000** [comment]
I have most of the research in at this point.
Currently working in setting up the clusters, facing some issues for which i am communicating with crw team. Waiting on their reply on my log paste.

**Comment by mohammedzee1000** [comment]
The primary acceptance criteria of this issue is met, closing in favour of new issue to track next steps
/close
